fedn/network/combiner/aggregators/aggregatorbase.py

```diff
@@ -86,9 +86,11 @@ def _validate_model_update(self, model_update):
         :return: True if the model update is valid, False otherwise.
         :rtype: bool
         """
-        data = json.loads(model_update.meta)["training_metadata"]
-        if "num_examples" not in data.keys():
-            logger.error("AGGREGATOR({}): Model validation failed, num_examples missing in metadata.".format(self.name))
+        try:
+            data = json.loads(model_update.meta)["training_metadata"]
+            num_examples = data["num_examples"]
+        except KeyError as e:
```

Comment:
I think we should be explicit and print out when "num_examples" are missing (since it is the only mandatory piece of metadata currently. Also, could in theory the keyerror be raised because  "training_metadata" is missing?